extension/src/popup/components/account/AccountHeader/index.tsx

```diff
@@ -54,76 +70,142 @@ export const AccountHeader = ({
+            <Link
+              className="AccountHeader__account-list-item__link"
+              to={ROUTES.importAccount}
+            >
+              <div className="AccountHeader__account-list-item__icon">
+                <Icon.Download />
+              </div>
+              <span className="AccountHeader__account-list-item__link-copy">
+                {t("Import a Stellar secret key")}
+              </span>
+            </Link>
+          </li>
+          <li className="AccountHeader__account-list-item">
+            <Link
+              className="AccountHeader__account-list-item__link"
+              to={ROUTES.connectWallet}
+            >
+              <div className="AccountHeader__account-list-item__icon">
+                <img
+                  className="AccountHeader__img-cube-icon"
+                  src={IconCube}
+                  alt=""
+                />
+              </div>
+              <span className="AccountHeader__account-list-item__link-copy">
+                Connect a hardware wallet
+              </span>
+            </Link>
+          </li>
+        </ul>
+      </AccountHeaderModal>
+      <AccountHeaderModal
+        isDropdownOpen={isNetworkSelectorOpen}
+        maxHeight={networksModalHeight.current}
+      >
+        <>
+          <div className="AccountHeader__network-selector">
+            {networksList.map((n, i) => (
+              <div
+                className="AccountHeader__network-selector__row"
+                key={n.networkName}
+                onClick={() =>
+                  dispatch(changeNetwork({ networkName: n.networkName }))
+                }
+              >
+                <div
+                  className="AccountHeader__network-icon"
+                  style={{ background: getNetworkColor(i) }}
+                />
+                <div className="AccountHeader__network-copy">
+                  {n.networkName}
+                </div>
+                {isActiveNetwork(n, networkDetails) ? (
+                  <div className="AccountHeader__network-selector__check">
+                    <Icon.Check />
+                  </div>
+                ) : null}
+              </div>
+            ))}
+          </div>
+
+          <hr className="AccountHeader__list-divider" />
+          <div className="AccountHeader__account-list-item">
+            <Link
+              className="AccountHeader__account-list-item__link"
+              to={ROUTES.addNetwork}
+            >
+              <div className="AccountHeader__account-list-item__icon">
+                <Icon.PlusCircle />
+              </div>
+              <span className="AccountHeader__account-list-item__link-copy">
+                {t("Add custom network")}
+              </span>
+            </Link>
+          </div>
+          <div className="AccountHeader__account-list-item">
+            <Link
+              className="AccountHeader__account-list-item__link"
+              to={ROUTES.addNetwork}
```

Comment:
should this be going to a different URL than the "Add custom network" link above?

Reply:
Yeah, this view doesn't exist yet. It'll link to the right place in the next PR